File: GameLayers/exhaustAttackCardsLayer.js
export class DefaultExhaustAttackCardsLayer {
    execute(world) {
    }
}

export class DiscardCardsExhaustAttackCardsLayer {
    execute(world) {
        world.attackCards.forEach(card => {
            var index = world.heroHand.indexOf(card)
            if (index != -1) {
                world.heroHand.splice(index, 1)
            }
        });
    }
}

File: python/GameLayers/exhaustAttackCardsLayer.py
import config

class DefaultExhaustAttackCardsLayer:
    def execute(self, world):
        pass

class DiscardCardsExhaustAttackCardsLayer:

    def execute(self, world):
        for card in world.attackCards:
            index = world.heroHand.index(card)
            if index != -1:
                world.heroHand.pop(index)


